static/css/timetable/partials/search_bar.scss

```diff
@@ -363,30 +421,38 @@
 }
 
 .trans50 {
-  // This makes it semi transparent
-  background-color: $white-transparent-0;
-  border: 1px solid $white-transparent-0;
-  border-top-color: transparent;
-  box-shadow: none;
-
+  @include theme() {
```

Comment:
Included `theme` here even though I didn't use a variable because .trans50 updates the styles of other classes and I need the updated styles here to be in the same scope (if I didn't include theme, styles don't get updated)